server/src/main/java/io/druid/server/ClientQuerySegmentWalker.java

```diff
@@ -84,25 +82,13 @@ public ClientQuerySegmentWalker(
     final FinalizeResultsQueryRunner<T> baseRunner = new FinalizeResultsQueryRunner<T>(
         toolChest.postMergeQueryDecoration(
             toolChest.mergeResults(
-                new MetricsEmittingQueryRunner<T>(
```

Comment:
I see, the problem is that IntervalChunkingQueryRunner is not always used by default, so we lost those metrics when upgrading.